### PR TITLE
Add intake-esm recipe

### PR DESCRIPTION
**Pull request by andersy005**

### CODE REVIEW

**Comment by andersy005** [comment]
@conda-forge/help-python, this recipe is ready for review.

File: recipes/intake-cesm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+build: 
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  number: 0
+  noarch: python
+
+requirements:
+
+  host:
+    - python > 3.5
```
Comment (beckermr) [inline recipes/intake-cesm/meta.yaml]: Remove the space after the `>` symbol.

File: recipes/intake-cesm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    - pip
+
+  run:
+    - python > 3.5
```
Comment (beckermr) [inline recipes/intake-cesm/meta.yaml]: Ditto here.

**Comment by andersy005** [comment]
Thank you @beckermr!

**Comment by beckermr** [comment]
OFC! Happy to merge once the tests pass!

**Comment by mariusvniekerk** [comment]
I'd merge master into the PR to be sure

**Comment by andersy005** [comment]
@conda-forge/help-python, this recipe is ready!

**Pull request merged**